### PR TITLE
Update README: load method and cache file

### PR DESCRIPTION
**Pull request by robskrob**
@edlebert thank you so much for this gem.

This PR just adds to the README which I think could help some of your users. 
* method definition of load provided
* note on cache file creation

I also have a working example [here](https://github.com/robskrob/money-oxr-example). Let me know if you would like for me to set up another PR to introduce this example into your project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ Or install it yourself as:
 
 ## Usage
 
-With the gem, you do not need to manually add exchange rates. Calling load will
-load the rates from the cache file or download the rates from the API depending
+With the gem, you do not need to manually add exchange rates. Calling, `MoneyOXR::RatesStore#load` will
+either load the rates from the cache file or download the rates from the API and create the file depending
```
Comment (edlebert) [inline README.md]: Actually, I removed the explicit load call from the README since it's no longer required.

File: README.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ Money.default_bank = MoneyOXR::Bank.new(
   cache_path: 'config/oxr.json',
   max_age: 86400
 )
+
+MoneyOXR::RatesStore.new({
+  app_id: 'abcd1234',
+  cache_path: 'config/oxr.json',
+  max_age: 86400
+}).load
```
Comment (edlebert) [inline README.md]: I'm not sure why this is here.  The MoneyOXR::Bank creates its own rates store.

Comment (robskrob) [reply README.md]: @edlebert thank you for bringing this to my attention, and it looks like you are right. The load method is not necessary. In [my explorations](https://github.com/robskrob/money-oxr-example) with your gem I notice that `money-oxr` makes an API call whenever `Money#exchange_to` or `Money#get_rate` is called -- if the json cache is not available or if the cache is stale. I have not tested the stale part my self.

**Comment by robskrob** [comment]
I suppose my documentation is unnecessary. I will send another PR if I find anything else.

Thank you for reviewing this PR.

**Pull request closed without merging**